__tests__/app.test.js

```diff
@@ -78,4 +79,45 @@ describe("/api/articles/:article_id", () => {
         expect(res.body.msg).toBe('Oh no! Please enter a valid article ID!');
       });
   });
+})
+
+describe("/api/articles/:article_id/comments", () => {
+  test("GET - status 200 - Returns a comment with a valid article id", () => {
+    return request(app)
+    .get("/api/articles/9/comments")
+    .expect(200)
+    .then((res) => {
+      expect(res.body.comments).toBeSortedBy('created_at', {
+        descending: true
+      }),
+      res.body.comments.forEach((comment) => {
+        expect(comment).toEqual(
+          expect.objectContaining({
+            comment_id: expect.any(Number),
+            article_id: expect.any(Number),
+            votes: expect.any(Number),
+            author: expect.any(String),
+            body: expect.any(String),
+            created_at: expect.any(String)
+          })
+        )
+      })
+    })
+  })
```

Comment:
need a 200 test for no comments on an article

Reply:
Hi, thanks for the feedback! Could you just explain this one a little more please?